src/ansys/aedt/core/visualization/post/post_circuit.py

```diff
@@ -0,0 +1,582 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2021 - 2024 ANSYS, Inc. and/or its affiliates.
+# SPDX-License-Identifier: MIT
+#
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+"""
+This module contains these classes: `FieldPlot`, `PostProcessor`, and `SolutionData`.
+
+This module provides all functionalities for creating and editing plots in the 3D tools.
+
+"""
+from __future__ import absolute_import
+
+import warnings
+
+from ansys.aedt.core import generate_unique_name
+from ansys.aedt.core import pyaedt_function_handler
+from ansys.aedt.core.generic.constants import unit_converter
+from ansys.aedt.core.visualization.post.common import PostProcessorCommon
+
+try:
+    import pandas as pd
+except ImportError:
+    pd = None
+    warnings.warn(
+        "The Pandas module is required to run some functionalities of PostProcess.\n"
+        "Install with \n\npip install pandas"
+    )
+
+
+class PostProcessorCircuit(PostProcessorCommon, object):
```

Comment:
```suggestion
class PostProcessorCircuit(PostProcessorCommon):
```
Not sure `object` is required, is it ?